docs/example/assets/capact-diagram.svg

```diff
@@ -883,7 +883,7 @@
             <xhtml:div
```

Comment:
For the sake of consistency, please also update comment in `docs/feature/policies/action-policy.md` (line 19)

```
# Enforces the Helm RocketChat Implementation
```